core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -35,9 +35,9 @@ oppia.controller('SettingsTab', [
       explorationInitStateNameService, explorationParamSpecsService,
       changeListService, alertsService, explorationStatesService,
       explorationParamChangesService, explorationWarningsService,
-      CATEGORY_LIST, explorationAdvancedFeaturesService,
-      EXPLORATION_TITLE_INPUT_FOCUS_LABEL, userExplorationEmailsHandler,
-      EditableExplorationBackendApiService) {
+      explorationAdvancedFeaturesService, ALL_CATEGORIES,
```

Comment:
This was just a merge with develop which had changed dependencies. I merely added EditableExplorationBackendApiService